forte/data/ontology/top.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from dataclasses import dataclass
+from enum import IntEnum
```

Comment:
Make sure you add a dependency to https://pypi.org/project/enum/, to we can keep backward compatibility.

Reply:
it's a built-in python package

Reply:
Sorry I mean this link: https://pypi.org/project/enum34/, I think this `enum` is available after Py34

Reply:
need to resolve this